docs/en/getting-started/install.md

```diff
@@ -332,6 +332,19 @@ For production environments, it’s recommended to use the latest `stable`-versi
 
 To run ClickHouse inside Docker follow the guide on [Docker Hub](https://hub.docker.com/r/clickhouse/clickhouse-server/). Those images use official `deb` packages inside.
 
+### Deploy on Easypanel {#deploy-on-easypanel}
```

Comment:
I'm hesitant to add a dependency for `getting-started` experience. Maybe we should add it to the list of supported integrations? https://github.com/ClickHouse/clickhouse-docs/blob/main/docs/en/integrations/index.mdx

Reply:
Hello @mshustov, Thank you very much for the feedback. 
Of course you know, but Integrations seemed a bit irrelevant.
Because this solution allows you to run ClickHouse on your own server with one click. 
It does not provide any integration on ClickHouse.